labop/utils/helpers.py

```diff
@@ -1,54 +1,21 @@
 import difflib
 import logging
-from typing import Tuple
 
 import sbol3
 
-import labop
-
 logger = logging.getLogger(__file__)
 logger.setLevel(logging.INFO)
 
 
-def prepare_document(namespace="https://bioprotocols.org/demo/") -> sbol3.Document:
+def prepare_document(
```

Comment:
This method doesn't fit neatly with `Protocol.initialize_protocol`  So now there appear to be two divergent starting points for creating a new protocol:

Case 1:
```
doc = prepare_document('http://foo.org')
protocol = labop.Protocol()
doc.add(protocol)
```
Case 2:
```
protocol, doc = Protocol.initialize_protocol()
sbol3.set_namespace(''http://foo.org')
```
It's not necessarily a bad thing to have more than one way to accomplish something, but I'm confused about which one of the above is stylistically preferred.

Reply:
Coming soon in another PR